src/components/AppList/index.scss

```diff
@@ -1,25 +1,26 @@
 @import '~scss/vars';
 
-.appList {
-  a {
+
+.appList, .categoryApps {
+  overflow: hidden;
+  _zoom: 1;
+
+  > div {
     &:nth-child(3n+4) {
-      &>div {
-        margin-right: 0;
-      }
+      margin-right: 0;
     }
   }
-  .noData{
+}
+
+.appList {
+  .noData {
     padding: 20px 0;
     text-align: center;
     font-size: 12px;
     color: $N75;
   }
 }
-.categoryApps{
-  margin-top: -16px;
-}
 
-.appsRow {
-  display: flex;
-  justify-content: space-between;
+.categoryApps {
```

Comment:
simple way is using flex, and remove `justify-content`

Reply:
If using flex, 6 or more apps will show one line.

Reply:
OK